src/proto/mgmt/svc.proto

```diff
@@ -102,6 +102,14 @@ message FabricInterfaces {
 	repeated FabricInterface ifaces = 1;
 }
 
+message BuildInfo
```

Comment:
Not sure how these wound up in your PR... Bad merge from master?

Reply:
Weird. I'll fix this.